languages/python/exercises/concept/string-methods/.docs/introduction.md

```diff
@@ -0,0 +1,16 @@
+## String Methods
+
+A `str` in Python is a sequence of Unicode code points which
```

Comment:
This exercise has `strings` as a prerequisite. Looking at the `processing-logs` exercise that unlocks this concept, it looks like the first bit (sequence of Unicode code points) is already explained there, so it can be removed from this document. The sequence operations bit is useful and specific to this exercise.